plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestHiveMetastoreMetadataQueriesAccessOperations.java

```diff
@@ -100,12 +105,12 @@ public void testSelectSchemasWithLikeOverSchemaName()
     @Test(dataProviderClass = DataProviders.class, dataProvider = "trueFalse")
     public void testSelectTablesWithoutPredicate(boolean allTablesViewsImplemented)
     {
+        mockMetastore.setAllTablesViewsImplemented(allTablesViewsImplemented);
```

Comment:
since `mockMetastore` is created once, there should be a `@BeforeMethod` that resets this, so tests fully isolated